### PR TITLE
Return sub file if dryrun

### LINKED ISSUE #1: [Request] Have Job Submitter create the submit file when given --dryrun
Currently, when given `--dryrun`, the `job_submitter` will create the proper folders, bash scripts, output some summary and exit. It would be nice to also create the submition file before giving a summary and exiting, without submitting to htc of course.

It should be a simple change, maybe incorporating the following lines in the main function?
```python
subfile = htcutils.make_subfile(
    cwd, job_df, output_dir=output_dir, duration=flavour, **additional_htc_arguments
)
```

Those lines are currently only in the [`_run_htc`](https://github.com/pylhc/PyLHC/blob/58d060236281e7b08216e76263f719fa012bb0fb/pylhc/job_submitter.py#L379) function.

The reason I'd like this is that while `job_submitter` makes life easy, it does not provide ways to do some of the wacky complicated things we can see in the batch docs. Going for a dry run and changing / adding things to the submit file would be an easy way to toy around. @JoschD what do you think?

### PR DESCRIPTION
**Pull request by mihofer**
Fixes #1 and #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ pip install /path/to/submitter
 
 Or simply from the online master branch, which is stable:
 ```bash
-pip install git+https://github.com/pylhc/submitter.git#egg=pylhc
+pip install git+https://github.com/pylhc/submitter.git#egg=submitter
```
Comment (JoschD) [inline README.md:31]: `pylhc_submitter` (sry my bad)

**Review by JoschD — CHANGES_REQUESTED** [review]

**Review by JoschD — DISMISSED** [review]
The last 9 commits were okay

File: pylhc_submitter/htc/utils.py [hunk 2]
```diff
@@ -165,20 +168,24 @@ def write_bash(
...
-    exec_path = EXECUTEABLEPATH.get(executable, executable)
     shell_scripts = [None] * len(job_df.index)
     for idx, (jobid, job) in enumerate(job_df.iterrows()):
         job_dir = Path(job[COLUMN_JOB_DIRECTORY])
-        bash_file_name = f"{BASH_FILENAME}.{jobid}.sh"
+        bash_file_name = f"{BASH_FILENAME}.{jobid}.{'bat' if on_windows() else 'sh'}"
         jobfile = job_dir / bash_file_name
         LOG.debug(f"Writing bash-file {idx:d} '{jobfile}'.")
         with open(jobfile, "w") as f:
             f.write(f"{SHEBANG}\n")
```
Comment (mihofer) [inline pylhc_submitter/htc/utils.py:185]: is not necessary on windows?

Comment (JoschD) [reply pylhc_submitter/htc/utils.py:185]: No, but it's a comment anyway

**Comment by mihofer** [comment]
also should be fixing broken documentation

**Review by JoschD — APPROVED** [review]

**Pull request merged**